File: WhereTo/Models/Lokal.cs
using System.Text.Json.Serialization;

namespace WhereTo.Models
{
    public class Lokal
    {
        public string LokalID { get; set; }= $"lokal:{Guid.NewGuid().ToString()}";
        public string? Name { get; set; }
        public string? Lokacija { get; set; }
        public string? Vlasnik { get; set; }
        public string? RadnoVreme { get; set; }
        public string? Opis { get; set; }
        public List<string>? Dogadjaji { get; set; }
        public List<string>? Tagovi { get; set;}
    }
}

File: WhereTo/Client/lokal.js
export class Lokal
{
    constructor(lokalID,name,lokacija,vlasnik,radnoVreme,opis,dogadjaji,tagovi,email,password)
    {
        this.lokalID= lokalID;
        this.name=name;
        this.lokacija=lokacija;
        this.vlasnik=vlasnik;
        this.radnoVreme=radnoVreme;
        this.opis=opis;
        this.email=email;
        this.password=password;
        this.dogadjaji=dogadjaji;
        this.tagovi=tagovi;
    }
}